docs/how-to-onboard-a-repository.md

```diff
@@ -0,0 +1,36 @@
+How to onboard a repository to KubeVirt Prow
+============================================
+
+Prow needs to be able to act on your repository, thus it requires at least write access. For this to work you need to invite [kubevirt-bot](https://github.com/kubevirt-bot) to your repository. This is a GitHub account we use for all CI matters. Also a webhook needs to get added to your repository, so that it sends back signals to Prow. The webhook configuration management is [automated](https://github.com/kubevirt/project-infra/blob/cdbbd894dfa7b9d4ab01139c7119e8dd54dfcc04/github/ci/prow-deploy/tasks/deploy.yml#L53), see configuration [here](https://github.com/kubevirt/project-infra/blob/0216025282ce2edb3feb15cd601bab1fdd253084/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L584). _See below if you do not want Prow to manage the webhook._ 
```

Comment:
Hi @dhiller , the links to files, wouldn't be better to use a branch name than a commit SHA-1

Reply:
Hey, I replaced the shas with branch name, also I added a bit more documentation. PTAL, thanks!